Main.dart

```diff
@@ -0,0 +1,40 @@
+void main()
+{
+    Pessoa p1 = Pessoa("Pedro", 1.76, DateTime.parse("2003-06-12"));
+    print(p1.dados());
+
+}
+
+class Pessoa {
+
+    String _nome;
+    double _altura;
+    DateTime _niver;
+
+    Pessoa(String nome, double altura, DateTime niver) {
+        this._nome = nome;
+        this._altura = altura;
+        this._niver = niver;
+
+    }
+
+    String get nome{ return this._nome.toUpperCase(); }
+    double get altura{ return this._altura; }
```

Comment:
qdo é coisa simples assim, simplesmente pra vc retornar um valor, vc pode trocar essa linha por algo assim:
double get altura => this._altura;

faz o teste aí pra vc ver